### ISSUE TITLE
Adjusting how we use "diversity" in the vision

### ISSUE THREAD

**Issue by torgo** [w3c/AB-public#169]
As discussed in the vision breakout session at the Advisory Committee meting today, I'd like to suggest that we need to adjust the way we use the word "diversity" in this document. My main issue is that I feel the word has been watered down, specifically in the following bullet point under "operational principles":

> * Diversity: We believe in diversity and inclusion of participants from different geographical locations, cultures, languages, disabilities, gender identities, industries, organizational sizes, and more. In order to ensure W3C serves the needs of the entire Web user base, we also strive to broaden diversity and inclusion for our own participants.

My *suggestion* is that factors such as industries and organisational sizes, whilst important, do not belong in the same list as the other factors mentioned. In fact, as @wareid pointed out, the Code of Conduct already includes an agreed [definition of diversity](https://www.w3.org/policies/code-of-conduct/#dfn-diversity), which I'd like to suggest this document reference. We can then include wording to the effect that "we also value varied perspectives, including industry, organisational size, and other factors".

**Comment by fantasai** [comment]
Quoting that definition,

> Diversity is variation across a wide range of identities, lived experiences, abilities, and perspectives. 

That's the definition. The subsequent list is explicitly labelled "can include, but is not limited to", i.e. it is not exhaustive, and calling out additional factors that are particularly relevant to our context is imho totally reasonable and appropriate.

**Comment by chaals** [comment]
I agree with @fantasai here. I would also suggest that in the specific relevant context, the diversity of perspectives based on different industrial/employment/... backgrounds **is**  as important (neither more nor less) as that of other diversity in lived experience.

**Comment by cwilso** [comment]
Note to self: Suggestion: split out inclusion as a separate bullet, ref to CoC, people participating as their full selves; rename the other bullet as diversity of perspective.  Neither is a closed-ended list, each will list appropriate things.

**Comment by jenstrickland** [comment]
Appreciate the discussion here. Suggest an approach that references the usual diversity mention and then an "as well as" for the addition of industry and org size. When this is separated it retains the original lens of diversity while expanding to the addition.

Example:

Diversity: We believe in diversity and inclusion of participants from different geographical locations, cultures, languages, disabilities, gender identities, and more, as well as industries and organizational sizes. In order to ensure W3C serves the needs of the entire Web user base, we also strive to broaden diversity and inclusion for our own participants.

**Comment by cwilso** [comment]
Gah, did not mean for autoclose to close this without confirmation.  @torgo , could you review the new text at https://w3c.github.io/AB-public/Vision#diversity and confirm that this addresses your concern, so we can close this issue?

**Comment by torgo** [comment]
I'm good with that and happy to close on that basis.
